inception-recommendation/src/main/java/de/tudarmstadt/ukp/inception/recommendation/sidebar/LearningCurveChartPanel.java

```diff
@@ -208,12 +212,20 @@ private String createJSScript(String aDataColumns, String aChartType) throws IOE
             try {
                 Details detail = fromJsonString(Details.class, detailJson);
 
+                //do not include the scores from disabled recommenders
+                List<Recommender> recommenderIfActive = recommendationService.getRecommenderIfActive(model.getObject().getProject(), detail.layer, detail.tool);
+                if(recommenderIfActive.size()<=0)
```

Comment:
Use `isEmpty()` instead of a size comparison.

Reply:
Done